setup.py

```diff
@@ -8,7 +8,7 @@
 
 extras_require = {
     'tsfresh': ['featuretools-tsfresh-primitives >= 0.1.0'],
-    'update_checker': ['featuretools-update-checker >= 1.0.0'],
+    'update_checker': ['featuretools-update-checker >= 2.0.0'],
```

Comment:
should be `'update_checker': ['alteryx-open-src-update-checker >= 2.0.0']`

Reply:
Fixed